es7/2017-01/summary.md

```diff
@@ -0,0 +1,46 @@
+- [SharedArrayBuffer](https://github.com/tc39/ecmascript_sharedmem) reached Stage 4
```

Comment:
General suggestion: Make conclusion headers unique and reference them from the summary page, so people can easily reference the discussion.